templates/disclaimer.tmpl

```diff
@@ -3,20 +3,20 @@
 {% block title %}RDM Manufacturer Index - Disclaimer{% endblock %}
 
 {% block content %}
-<div style="padding: 5px">
-<b>Disclaimer</b>
+    <div style="padding: 5px">
+        <b>Disclaimer</b>
 
-<p>This site makes no guarantees about the correctness of the PID
-information.</p>
+        <p>This site makes no guarantees about the correctness of the PID
+            information.</p>
 
-<p>The authoritative source for Manufacturer IDs is the 
-  <a href="http://tsp.plasa.org/tsp/working_groups/CP/mfctrIDs.php">
-    PLASA Site</a>.</p>
+        <p>The authoritative source for Manufacturer IDs is the
+            <a href="http://tsp.plasa.org/tsp/working_groups/CP/mfctrIDs.php">
+                PLASA Site</a>.</p>
 
-<p>For PIDs in the ranges {0x0000, 0x7fff} and {0xffe0, 0xffff}, the E1.20 and
-E1.37-X documents are authoritative. These can be purchased from
-the PLASA
-<a href="http://tsp.plasa.org/tsp/documents/published_docs.php">
-  Technical Standards Program</a>.</p>
-</div>
+        <p>For PIDs in the ranges {0x0000, 0x7fff} and {0xffe0, 0xffff}, the E1.20 and
+            E1.37-X documents are authoritative. These can be purchased from
```

Comment:
s/purchased/downloaded/ it may also be worth adding a note thanking the insurance sponsor that provides them for free.